main/ertp/guide/other-concepts.md

```diff
@@ -7,4 +7,29 @@ concepts:
 
 * __E() and tildot (~.)__: Instead of `obj.foo()`, we can write `E(obj).foo()` or the syntactic sugar, `obj~.foo()` and get a promise for the result. The syntax means "deliver the message foo() to the actual object asynchronously, in its own turn, wherever and whenever it is, even if it is local." Using `E` or `~.`, you can talk asynchronously to local and remote objects in exactly the same way.
 
-* __Presences__: Presences are our name for the local object that stands in for the remote object. If `obj` is a presence of a remote object, we can send messages to the remote object by using "~." on `obj`, as in the above example.
\ No newline at end of file
+* __Presences__: Presences are our name for the local object that stands in for the remote object. If `obj` is a presence of a remote object, we can send messages to the remote object by using "~." on `obj`, as in the above example.
+
+
+## Security Properties
```

Comment:
I think we need to move this back into the original tutorial. We can explain that `depositAll` will error if the payment isn't a valid payment.

Reply:
:+1: